db/migrate/20160403055927_create_deployment_authentications.rb

```diff
@@ -0,0 +1,33 @@
+class CreateDeploymentAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :deployment_authentications do |t|
```

Comment:
I'm not familiar with ruby or the ansible code, but the `master-config.yaml` allows multiple identity providers to be configured at the same time. Does this handle that case?  Does it intend to?